### PR TITLE
fix(deps): we fixed some missing depen

### PR DESCRIPTION
**Pull request by Sanix-Darker**
## WHAT

While reviewing your project, we found that there was some missing depencies, so in this pull request
We decided to fix that, so in the future, a simple "pip install -r requirements.txt" should be enought

### CODE REVIEW

File: utilities/audio.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import moviepy.editor as mp
 
 def convert_to_wav(path):
-    output_path = "/voice.wav"
+    output_path = "/home/dk/Downloads/out-put-voice.wav"
```
Comment (Sanix-Darker) [inline utilities/audio.py:4]: just for Permission issues, this needed to be changed !

Comment (djoufson) [reply utilities/audio.py:4]: This one may not work, since this path does not exist in every computer

Comment (Sanix-Darker) [reply utilities/audio.py:4]: Yes, it was for my personnal testing !
That's why i said it was for "permission issues"
You can get rid of this change !

**Comment by Sanix-Darker** [comment]
This PR is jist for dependencies freeze !

**Comment by djoufson** [comment]
Thank you for contributing 👍🏾

**Pull request merged**